CodeStarterCamp_Week3/main.swift

```diff
@@ -1,12 +1,5 @@
-//
-//  main.swift
-//  CodeStarterCamp_Week3
-//
-//  Created by yagom.
-//  Copyright © yagom academy. All rights reserved.
-//
-
 import Foundation
 
-print("Hello, World!")
-
+let misterLee: Person = Person(money: 5000)
+let misKim: Person = Person(money: 10000)
+let yagomBucks: CoffeeShop = CoffeeShop(sales: 500000, barista: misterLee)
```

Comment:
타입의 초기화를 해서, 상수에 잘 할당해주셨네요! 😄 

팁을 하나 드리자면
`=` 오른 쪽에 이미 눈에 띄게 대문자로 시작하는 `Person(~` 또는 `CoffeeShop(~` 표현이 있잖아요?
그래서 굳이 상수 오른쪽에 `콜론(:)` 붙여서, 명시적으로 `Type annotation` 을 해주지 않아도 괜찮아요.

지워주면 오히려 깔끔하고 가독성도 좋아져요.
중복 정보를 없애는 기분도 들구요 ㅎㅎㅎ

+ 그리구 misKim → `missKim` (미스 김)으로 수정 필요해보입니다! 👍🏻 

```suggestion
let misterLee = Person(money: 5000)
let missKim = Person(money: 10000)
let yagomBucks = CoffeeShop(sales: 500000, barista: misterLee)
```